makefile

```diff
@@ -0,0 +1,59 @@
+#REQUIREMENTS BEFORE RUNNING MAKE ALL: ruby, yarn, docker, graphviz, imagemagick, nvm, postgres
+.PHONY: test
+
+all: install_dependencies docker setup_env setup_db seed_db create_user run
+
+test_all: update test
+
+docker: setup_docker_network setup_docker_container setup_env
+
+run:
+	docker start placecal-db || true
+	bin/dev
+
+down:
+	docker stop placecal-db
+
+# does not remove postgres image, just placecal container & network
+clean:
+	docker stop placecal-db || true
+	docker container rm placecal-db || true
+	docker network rm placecal-network
+
+install_dependencies:
+	yarn
+	bundle install
+
+setup_docker_network:
+	docker network create placecal-network
+
+setup_docker_container:
+	docker create --name placecal-db --network placecal-network --network-alias postgres -p 5432:5432 --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5 -e 'POSTGRES_DB=placecal_dev' -e 'POSTGRES_USER=postgres' -e 'PGUSER=postgres' -e 'POSTGRES_PASSWORD=foobar' -e 'POSTGRES_PORT=5432' postgres:14.1
+	docker start placecal-db 
+
+setup_env:
+	cp .env.example .env
+	echo POSTGRES_HOST=localhost >> .env
+	echo POSTGRES_USER=postgres >> .env
+	echo PGPASSWORD=foobar >> .env
+	echo PGHOST=localhost >> .env
+	echo PGUSER=postgres >> .env
+
+setup_db:
+	bundle exec rails db:setup db:migrate db:seed
+	bundle exec rails import:all_events
+
+seed_db:
+	rails db:dump_production_and_restore_other restore_on_local=1
+
+create_user:
```

Comment:
This command duplicates behaviour from the `rails db:seed`  which also [creates an user](https://github.com/geeksforsocialchange/PlaceCal/blob/292638633b3604dd7ed5043e229165ae0009e05e/db/seeds/001_users.rb#L7).

Reply:
ah! Will remove `db:seed` from the db setup